intro-to-ruby.rb

```diff
@@ -1,14 +1,17 @@
 #create a string and store it in a variable
-
+stuff= "Hello world"
+def variable(stuff)
+    "#{stuff}! I'm using ruby."
+end
 #print out the value of the variable you just created
-
+puts variable(stuff)
```

Comment:
careful. Since you're specifying `stuff` is a string on line 4, you need to pass it as a string. This would throw an error.